tests/snappi_tests/multidut/pfc/files/m2o_fluctuating_lossless_helper.py

```diff
@@ -345,6 +340,11 @@ def __gen_data_flow(testbed_config,
         elif 'Test Flow 2 -> 0' in flow.name:
             eth.pfc_queue.value = pfcQueueValueDict[flow_prio[1]]
 
+    src_port = 5000 + eth.pfc_queue.value * 1000
```

Comment:
1. can you use a global variable for the starting port `5000`
2. I would prefer to remove the *1000 steps, just use `5000 + eth.pfc_queue.value`.  For two flows, port 5000 and 6000 may still hash to the same port, but 5000 and 5001 are unlikely to be hashed to same port in most platform.